gui/wxpython/lmgr/layertree.py

```diff
@@ -491,7 +491,7 @@ def OnLayerContextMenu(self, event):
         if ltype not in ("group", "command"):
             if numSelected == 1:
                 self.popupMenu.AppendSeparator()
-                if ltype != "raster_3d":
+                if ltype != "raster_3d" and not self.mapdisplay.IsPaneShown("3d"):
```

Comment:
This is just a suggestion, but this may be more understandable:

```suggestion
                if not (ltype == "raster_3d" or self.mapdisplay.IsPaneShown("3d")):
```

Reply:
Thanks.